api/controllers/CommentController.js

```diff
@@ -172,5 +173,45 @@ module.exports = {
       })
       .then(res.ok, res.serverError)
     })
+  },
+
+  createBatchFromEmailForm: function (req, res) {
+    try {
+      var tokenData = Email.decodeFormToken(req.param('token'))
+    } catch (e) {
+      return Promise.resolve(res.serverError(new Error('Invalid token: ' + req.param('token'))))
+    }
+
+    const { communityId, userId } = tokenData
+
+    const replyText = postId => req.param(`post-${postId}`)
```

Comment:
`createAndPresentComment` expects to get reply text that is properly formatted HTML (which it does from TinyMCE), otherwise it's not going to display correctly on the site and in email notifications. the simplest way is to parse the text as markdown; see e.g. `Comment.cleanEmailText`